src/tagmanager.cpp

```diff
@@ -127,31 +128,26 @@ int TagManager::tag_add_command(Input input, Output output) {
     return 0;
 }
 
-int TagManager::removeTag(Input input, Output output) {
-    string tagNameToRemove;
-    if (!(input >> tagNameToRemove)) {
-        return HERBST_NEED_MORE_ARGS;
-    }
-    auto targetTagName = input.empty() ? get_current_monitor()->tag->name : input.front();
-
-    auto targetTag = find(targetTagName);
-    auto tagToRemove = find(tagNameToRemove);
-
-    if (!tagToRemove) {
-        output << input.command() << ": Tag \"" << tagNameToRemove << "\" not found\n";
-        return HERBST_INVALID_ARGUMENT;
-    }
-
-    if (!targetTag) {
-        output << input.command() << ": Tag \"" << targetTagName << "\" not found\n";
-        return HERBST_INVALID_ARGUMENT;
-    }
+void TagManager::removeTagCommand(CallOrComplete invoc) {
```

Comment:
Why is the method called removeTagCommand when the command is actually called merge tag?

Reply:
This happend to be historically, but you're right, I should rename it to something like  `mergeTagCommand`, shouldn't I?

Reply:
It would make sense to do it now, while you're moving stuff in here anyway, yes :)

I would approve of a renaming. It confused me quite a bit, while reading the commit message subject
and thinking "wait a minute, nothing in the diff says "merge" :D

Reply:
(I've renamed it)

Reply:
Cool! Thank you :)